Aplib.Core/Intent/Tactics/AnyOfTactic.cs

```diff
@@ -1,56 +1,56 @@
-﻿using System;
+﻿using Aplib.Core.Belief;
+using Aplib.Core.Intent.Actions;
 using System.Collections.Generic;
-using Action = Aplib.Core.Intent.Actions.Action;
 
 namespace Aplib.Core.Intent.Tactics
 {
     /// <summary>
     /// Represents a tactic that executes any of the provided sub-tactics.
     /// </summary>
-    public class AnyOfTactic : Tactic
+    public class AnyOfTactic<TBeliefSet> : Tactic<TBeliefSet> where TBeliefSet : IBeliefSet
```

Comment:
Laten we de where altijd op een nieuwe lijn doen:
```suggestion
    public class AnyOfTactic<TBeliefSet> : Tactic<TBeliefSet> 
        where TBeliefSet : IBeliefSet
```